apps/studio/components/interfaces/APIKeys/QuickKeyCopy.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { useState } from 'react'
+import { Button, EyeIcon, EyeOffIcon, Skeleton, WarningIcon } from 'ui'
+import { ChevronsUpDownIcon } from 'lucide-react'
+import { useParams } from 'common'
+import { useAPIKeysQuery } from 'data/api-keys/api-keys-query'
+import { SimpleCodeBlock } from 'ui/src/components/SimpleCodeBlock'
+import { useCheckPermissions, usePermissionsLoaded } from 'hooks/misc/useCheckPermissions'
+import { PermissionAction } from '@supabase/shared-types/out/constants'
+import { useProjectSettingsV2Query } from 'data/config/project-settings-v2-query'
+
+const frameworkOptions = ['React', 'Vue', 'Angular', 'Svelte'] // Add more as needed
+
+const QuickKeyCopyWrapper = () => {
+  const [selectedFramework, setSelectedFramework] = useState('React')
+
+  return (
+    <div className="flex flex-col gap-0 bg-alternative/50 border rounded-lg overflow-hidden">
+      <div className="flex items-center gap-3 text-xs justify-between px-5 py-4">
+        <div className="flex flex-col gap-0">
+          <h4 className="text-sm">Quick key copy</h4>
+          <p className="text-foreground-lighter">
+            Choose your framework and paste the code into your environment file.
+          </p>
+        </div>
+        <Button
```

Comment:
The UX on this button is quite awkward imo - i was expecting a dropdown because of the chevrons but it just swaps the value
![image](https://github.com/user-attachments/assets/1d2be2e1-014b-42b4-a542-241a656d3245)

Reply:
Happy to park this for follow up iteration, marking resolved